test/spec/lzwCompress.spec.js

```diff
@@ -76,6 +76,12 @@ describe('lzwCompress Tests', function () {
       expect(lzwCompress.unpack(lzwCompress.pack('"{\\"abnabnabn\\":\\"asd\\"}"'))).toEqual('"{\\"abnabnabn\\":\\"asd\\"}"'); // issue #1 - https://github.com/floydpink/lzwCompress.js/issues/1
     });
 
+    it('should pack/unpack objects with javascript while avoiding prototype methods', function() {
+      var jsString = 'toString,toString,toString,toString,toString,toString,toString,toString,toString,toString,toString,toString,toString,toString,toString,toString,toString,toString,';
```

Comment:
I just played around with this test case and realized even when one `toString` is removed, it passes with the old code.

I am just curious - do we know why we need these many `toString` occurrences? 🙂

Reply:
So I'm not an expert on compression by any stretch of the imagination, but I assume something in the compression algorithm only chooses larger substring chunks if the substring is present enough times? With less instances of `toString`, the dictionary ends up having `toStrin`, `toStri`, etc, instead of the full `toString`. 
I was personally running into this problem on a very large (~1mb), deeply nested object that happened to contain javascript snippets - there it was far less prevalent than in this string, so I suspect there's something going on with how the dictionary increases in size over time and how it determines what substrings to map to? Anyway, all conjecture on my part there :P

Reply:
yeah - that makes sense!

I had put this library together back in 2012 when I had a need to make sure large JSON objects stored within IndexedDB/WebSQL had to brought within the then permissible limits of a few megs. pretty sure the lzw compression algorithm is what I copy/pasted from wikipedia or somewhere :)